pyproject.toml

```diff
@@ -76,6 +77,7 @@ build-backend = "poetry.core.masonry.api"
 filterwarnings = [
     "error",
     "ignore:You are encoding a BooleanParameterType:UserWarning",
+    "ignore:Number of bits parsed:UserWarning"
```

Comment:
I think this is already covered by the individual `pytest.mark.filterwarnings()` on main to limit the filters a bit more. Or is there added things in this test that cover that? I worry about avoiding these so we get into adding more incorrect bit-length XTCEs without realizing it in the future.
```suggestion
```